jdaviz/configs/default/plugins/subset_plugin/subset_plugin.py

```diff
@@ -94,7 +101,7 @@ def _sync_selected_from_state(self, *args):
 
     def _on_subset_update(self, *args):
         self._sync_selected_from_state(*args)
-        if self.subset_selected == 'Create New':
+        if 'Create New' in self.subset_selected:
```

Comment:
this could give us problems if we ever support subset renaming [:cat:](https://jira.stsci.edu/browse/JDAT-2350). (I don't think there's anything needed to be done here for now, just something to keep in mind for later)